src/main/java/seedu/address/model/travelplan/AccommodationList.java

```diff
@@ -128,4 +128,8 @@ public boolean equals(Object other) {
     public int hashCode() {
         return accommodations.hashCode();
     }
+
+    public UniqueAccommodationList getModifiableAccommodationList() {
```

Comment:
This one i not sure tho. I'm thinking to sync everything to use AccommodationList XXXList instead of UniqueXXXList